### PR TITLE
Fix bug in predict in examples.

### LINKED ISSUE #258: Bug in predict in the xgboost example?
https://github.com/kubeflow/fairing/blob/08c81437fd9ed720cf4b780e25097fef9fa16cd6/examples/prediction/xgboost-high-level-apis.ipynb#L175

This line seems wrong

```
return [[prediction.item(0), prediction.item(0)]]
```

Why would we want to return the same value twice? I suspect it should be

```
return [[prediction.item(0), prediction.item(1)]]
```

### PR DESCRIPTION
**Pull request by abcdefgs0324**
**What this PR does / why we need it**:
Fix bugs in the function `predict` in examples

**Which issue(s) this PR fixes**:
Fixes #258 

/assign @jlewi 
/assign @zhenghuiwang

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/335)

### CODE REVIEW

**Comment by abcdefgs0324** [comment]
Also fix #202

**Comment by abcdefgs0324** [comment]
Fix #202

**Comment by jlewi** [comment]
/lgtm
/approve

Thanks for the bug fix!

**Comment by abhi-g** [comment]
@abcdefgs0324 please resolve integration test failure

File: examples/prediction/HousingServe.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def __init__(self, model_file=MODEL_FILE):
   def predict(self, X, feature_names):
     """Predict using the model for given ndarray."""
     prediction = self.model.predict(data=X)
-    return [[prediction.item(0), prediction.item(0)]]
+    return [[prediction.item(0), prediction.item(1)]]
```
Comment (abhi-g) [inline examples/prediction/HousingServe.py]: @karthikv2k Is this change correct? Its possible that the prediction.item(0) was intentional as the prediction is returning a singleton, but the API requires a calls and confidence?

Comment (karthikv2k) [reply examples/prediction/HousingServe.py]: should be returning all the results for the input X - something like return [prediction]

Comment (abcdefgs0324) [reply examples/prediction/HousingServe.py]: Got it, thanks!

**Comment by jinchihe** [comment]
@abcdefgs0324 Pls rebase, the integration test problem should have been fixed in #337 thanks.

**Comment by abcdefgs0324** [comment]
@jinchihe Rebased, but it still fails.

**Comment by jinchihe** [comment]
@abcdefgs0324 That's strange,  same reason. that's caused the old version of `tornado`, but in the #337 the `tornado` should has been upgraded. Need more check...

```
ImportError: cannot import name 'TracebackFuture' from 'tornado.concurrent'
```

**Comment by jinchihe** [comment]
@abcdefgs0324 Can you try that by adding one line to upgrade tornado in the [Dockerfile.test](https://github.com/kubeflow/fairing/blob/master/Dockerfile.test) file?

```
pip install -U tornado
```

File: Dockerfile.test [hunk 2]
```diff
@@ -24,6 +24,8 @@ RUN pip install -r examples/prediction/requirements.txt
 COPY setup.py requirements.txt /opt/fairing/
 RUN pip install -e .
 
+pip install -U tornado
```
Comment (jinchihe) [inline Dockerfile.test]: Should be "RUN pip install -U tornado", thanks

**Comment by abcdefgs0324** [comment]
@jinchihe Solved finally. Thanks for your help!

**Comment by jinchihe** [comment]
@abcdefgs0324 Thank you for your persistence!

/lgtm

**Pull request merged**